static/js/admin/src/AddMenu.js

```diff
@@ -8,17 +8,23 @@ define('AddMenu', [ 'jquery', 'jquery-ui', 'underscore', 'CreateContainerForm',
 	};
 	
 	AddMenu.prototype.getMenuItems = function() {
+		var canIngest = $.inArray('ingest', this.container.permissions) !== -1;
 		var items = {};
 
-		if ($.inArray('ingest', this.container.permissions) != -1) {
+		if (canIngest) {
 			var self = this;
 
-			items["addContainer"] = {
+			if ((canIngest && (this.container.type === "Folder" || this.container.type === "Collection")) ||
+				($.inArray('createCollection', this.container.permissions) !== -1 && this.container.type === "AdminUnit") ||
+				($.inArray('createAdminUnit', this.container.permissions) !== -1 && this.container.type === "ContentRoot")
+			) {
+				items["addContainer"] = {
 				name : "Add Container",
```

Comment:
My inclination would be to say "Add Collection", "Add Folder", "Add Admin Unit" here